README.md

```diff
@@ -1 +1,63 @@
-# PokemonAPI
\ No newline at end of file
+# Pokémon API
+> This web app displays different Pokémon gotten from the RESTful [Pokémon API](https://pokeapi.co/) in which you can like your favorites. It also allows you to see further information about each Pokémon and leave a comment. All data is recorded thanks to the [Involvement API service](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270). Built with Javascript.
+
+## 🛠️ Built With
+
+- HTML, CSS & Javascript
+- Webpack
+- Jest
+- Babel
+- Linters: Lighthouse, Webhint, Stylelint & ESlint
+
+## 📹 [Video documentation](https://www.loom.com/share/febaae743e004d9aa35818b252fdca95)
+
+## 🧮 Prerequisites
+
+### Setup
+
+Follow these steps on your console to properly clone this repository on your desktop:
+
+```
+$ cd desktop
+$ git clone 'repo_path'
+$ cd 'repo_name'
+$ code . 
+Run 'npm i' to install the dev dependencies.
+Run 'npm start' to run the website from your local machine.
+Run 'npm run build' to build the code after editing it.
+```
+
+### Install
+
+- [Webpack](https://webpack.js.org/guides/getting-started/)
+- [Jest](https://jestjs.io/docs/getting-started)
+
+## 👤 Authors
+
+### Mike Martínez
+
+- GitHub: [@mikemtzp](https://github.com/mikemtzp)
+- Twitter: [@mikemtzp](https://twitter.com/mikemtzp)
+- LinkedIn: [Miguel Martínez Prieto](https://www.linkedin.com/in/miguel-mart%C3%ADnez-prieto-a42406166/)
+
+### Menanya Morris
+- GitHub: [@Emmykage](https://github.com/Emmykage)
+- Twitter: [@emmykage](https://twitter.com/emmykage)
+- LinkedIn: [Menanya Morris](https://www.linkedin.com/in/morris-menanya-a51985104/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/mikemtzp/PokemonAPI/issues).
+
+## ⭐️ Show your support
+
+Give a ⭐️ if you like this project!
+
+## 🥇 Acknowledgments
+
+- This README.md was elaborated with the [Microverse readme-template](https://github.com/microverseinc/readme-template)
+
+## 📝 License
+
+[MIT License](https://github.com/mikemtzp/PokemonAPI/blob/master/MIT.md)
```

Comment:
- [ ]  To make a professional readme file kindly fix the broken link.

Reply:
@ahmetbozaci Link fixed, thanks for noting it.😁